.github/workflows/ci.yml

```diff
@@ -61,7 +61,7 @@ jobs:
         run: just services
 
       - name: Load the data into the database
-        run: just load
+        run: just ingest_load
```

Comment:
The gRPC ingest is not used in the workflow, correct?
How de we ensure it keeps working?
Or is this part of the workflow refactor?

Reply:
I would prefer to separate `on push` tests to test only the service changed. So changes to for example `ingest` would launch `ingest-unit` and `ingest-load` with `integration`. Changes to `api` would launch `unit`, `load` and `integration`. `on pull_request`. But that is part of the workflow refactor.

Reply:
Ok, let's get back to this with the workflow refactor.